indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -37,43 +37,97 @@
                             'negative': _("No reviewer proposed to accept this abstract"),
                             'mixed': _("Not all reviewers proposed to accept this abstract")} %}
     {% set state = abstract.get_track_reviewing_state(track) %}
-    <span class="i-tag outline {{ mapping_class[state.name] }}" title="{{ mapping_title[state.name] }}">
-        {{ state.title -}}
-    </span>
+    <div class="state-icon {{ mapping_class[state.name] }} flexrow f-a-center f-j-center"
+         title="{{ mapping_title[state.name] }}">
+        <span
+            {%- if state.name in ['positive', 'conflicting'] %}
+                class="icon-checkmark"
+            {% elif state.name in ['mixed', 'negative'] %}
+                class="icon-cross"
+            {%- endif -%}
+        ></span>
+    </div>
```

Comment:
Such an icon would be more suitable for the *Mixed* state imo:
![image](https://cloud.githubusercontent.com/assets/1579899/20521721/e6307fb0-b0ac-11e6-95c8-cbcdf7ca47b7.png)